gapic/templates/tests/unit/gapic/%name_%version/%sub/test_macros.j2

```diff
@@ -119,13 +140,121 @@ def test_{{ method_name }}_empty_call():
         {% if method.client_streaming %}
         assert next(args[0]) == request
         {% else %}
+        {% with method_settings = api.all_method_settings.get(method.meta.address.proto) %}
+        {% if method_settings is not none %}
+        {% for auto_populated_field in method_settings.auto_populated_fields %}
+        # Ensure that the uuid4 field is set according to AIP 4235
+        assert re.match(r"[a-f0-9]{8}-?[a-f0-9]{4}-?4[a-f0-9]{3}-?[89ab][a-f0-9]{3}-?[a-f0-9]{12}", args[0].{{ auto_populated_field }})
```

Comment:
Ditto previous comment: We have this check in multiple places, so I suggest having this in a macro.

Reply:
Fixed in https://github.com/googleapis/gapic-generator-python/pull/1985/commits/70f0c64b54bb0551295dbf7f23ced7507202044c